examples/resources/plugins/teleport-mattermost-cloud.toml

```diff
@@ -5,16 +5,11 @@ identity = "/var/lib/teleport/plugins/mattermost/auth.pem"   # Identity file pat
 
 [mattermost]
 url = "https://mattermost.example.com" # Mattermost Server URL
-team = "team-name"                     # Mattermost team in which the channel resides.
-channel = "channel-name"               # Mattermost Channel name to post requests to
 token = "api-token"                    # Mattermost Bot OAuth token
-secret = "signing-secret-value"        # Mattermost API signing Secret
-
-[http]
-public_addr = "example.com" # URL on which callback server is accessible externally, e.g. [https://]teleport-mattermost.example.com
-# listen_addr = ":8081" # Network address in format [addr]:port on which callback server listens, e.g. 0.0.0.0:443
-https_key_file = "/var/lib/teleport/plugins/mattermost/server.key"  # TLS private key
-https_cert_file = "/var/lib/teleport/plugins/mattermost/server.crt" # TLS certificate
+recipients = [
```

Comment:
Are these changes to the Mattermost example configuration intentional? I don't see how they release to PagerDury.

Reply:
These were changes I neglected to reset when I changed the PR base from my Mattermost branch to `master`. I left them in because, when I merged the Mattermost branch, I had to copy the content of these files to the Mattermost guide and leave the files themselves unchanged. This was because, at the time, the `*toml` changes were triggering our GitHub Actions automation for non-docs changes. 

I'll reset these changes so we have a cleaner history here.